### ISSUE TITLE
Change to pip3 for Ubuntu 20

### ISSUE THREAD

**Issue by JJJ** [timnolte/neovim-ide-setup#1]
https://github.com/timnolte/neovim-ide-setup/blob/10f203f5c25b2df98efd7c3312beab6711333baf/setup.sh#L33

Will need to be `pip3` instead of `pip`.

**Comment by timnolte** [comment]
Yeah, so I usually setup `pip` to point to `pip3`, although I thought this was the default on Ubuntu 20. I thought that Python 3 was the default for Ubuntu 20 when installing the standard Python package.

**Comment by JJJ** [comment]
I did consider aliasing it. Just figured I would document everything I ran into.

I did:

```
sudo apt install python3-pip
```

Which did not give me `pip` but did give me `pip3`.

**Comment by timnolte** [comment]
Cool, yeah, and as I was looking through my she'll setup I did see that I have been setting up `pip3` as the alternative to `pip`.
